addons/website_event/static/src/scss/event_templates_list.scss

```diff
@@ -21,15 +21,15 @@
 .o_wevent_index {
     // Events List
     .o_wevent_events_list {
-        .opt_events_list_columns {
+        #o_wevent_event_grid_layout {
```

Comment:
We tend to avoid ids, if we go with the t-call for each template, should we replace them with selectors?